src/uu/cp/src/cp.rs

```diff
@@ -1742,41 +1757,89 @@ fn handle_existing_dest(
             backup_dest(dest, &backup_path, is_dest_removed)?;
         }
     }
-    if !is_dest_removed {
+
+    let dest_was_removed = if is_dest_removed {
+        // TODO
```

Comment:
why this todo ?

Reply:
That was an uncommon edge case, but I'll have to refresh my memory on the details. We don't have tests covering it.